onnx/common/ir.h

```diff
@@ -1250,5 +1300,23 @@ inline const_graph_node_list_iterator Node::reverseIterator() const {
   return iterator().reverse();
 }
 
+inline const use_list Value::uses() const {
+  use_list all_uses = uses_in_current_graph_;
+  owningGraph()->forEachNode([this, &all_uses](const Node* node) {
+    if (node->owningGraph() == this->owningGraph()) {
+      // skip non-subgraph
+      return;
+    }
+    if (node->kind() == kCaptured) {
```

Comment:
The names from outer scope are imported as a `kCaptured` node, https://github.com/onnx/onnx/blob/master/onnx/common/ir_pb_converter.cc#L277